spec/features/user_contributes_new_councillor_spec.rb

```diff
@@ -18,6 +18,30 @@
       end
     end
 
+    context "when a person submit an invalid email" do
+      before :each do
+        visit new_authority_councillor_contribution_path(authority.short_name_encoded)
+
+        within ".councillor-contribution-councillors fieldset" do
+          fill_in "Full name", with: "Mila Gilic"
+          fill_in "Email", with: "mgilic.invalid"
+        end
+
+        click_button "Submit"
+      end
+
+      it "displays an error message" do
+        expect(page).to have_content("does not appear to be a valid e-mail address")
+      end
+
+      # FIXME: This test is passing but it's not testing what we want
```

Comment:
This test is supposed to make sure the entry with an invalid email address isn't added to the list of councillors we're about to submit. The test passes but it's not testing what we want it to test.